Source/WebCore/css/CSSToStyleMap.cpp

```diff
@@ -537,13 +537,19 @@ void CSSToStyleMap::mapNinePieceImageSlice(const CSSValue& value, NinePieceImage
 void CSSToStyleMap::mapNinePieceImageSlice(const CSSBorderImageSliceValue& value, NinePieceImage& image)
 {
     // Set up a length box to represent our image slices.
-    auto side = [](const CSSPrimitiveValue& value) -> Length {
+    auto side = [](const CSSPrimitiveValue& value, const CSSToLengthConversionData& conversionData) -> Length {
```

Comment:
Why not capture the conversion data instead of passing it, to keep the `setImageSlices` call easy to read?